cluster/manifests/visibility/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
I think you need to name this file `01-namespace.yaml` or something like that, to ensure CLM creates the namespace before creating resources in it. :X

Reply:
The directory needs to be renamed as well, otherwise logging-agent change above would fail.